pika/adapters/select_connection.py

```diff
@@ -155,32 +157,34 @@ def _get_poller(self):
         return poller
 
 
-class SelectPoller(object):
-    """Default behavior is to use Select since it's the widest supported and has
-    all of the methods we need for child classes as well. One should only need
-    to override the update_handler and start methods for additional types.
+class _PollerBase(object):  # pylint: disable=R0902
+    """Base class for select-based IOLoop implementations"""
+    __metaclass__ = abc.ABCMeta
```

Comment:
@gst, thank you for pointing this out. I had no idea about this change in Python v3.x.